python/samba/tests/source.py

```diff
@@ -60,7 +60,7 @@ def get_source_file_contents():
     """Iterate over the contents of all python files."""
     for fname in get_python_source_files():
         try:
-            f = open(fname, 'rb')
+            f = open(fname, 'r')
```

Comment:
Somehow the `samba.tests.source.python3` test passed on my local env, but failed in gitlab ci autobuild env. Not sure how it happened yet. Maybe skip this commit for now.